src/app/pages/vm/vm-wizard/vm-wizard.component.ts

```diff
@@ -427,6 +454,26 @@ export class VMWizardComponent {
       const vcpu_limit = _.find(this.wizardConfig[1].fieldConfig, {'name' : 'vcpu_limit'});
       vcpu_limit.paraText = helptext.vcpus_warning + ` ${this.maxVCPUs} ` + helptext.vcpus_warning_b;
     })
+
+    this.ws.call("device.gpu_pci_ids_choices").subscribe((pci_choices: Object) => {
+      const isolated_gpu_pci_ids_conf = _.find(this.wizardConfig[5].fieldConfig, {name : "isolated_gpu_pci_ids"});
+      for(let key in pci_choices) {
+        isolated_gpu_pci_ids_conf.options.push({label: key, value: pci_choices[key]})
+      }
+    })
+
+    this.ws.call("device.get_gpus").subscribe((gpus) => {
```

Comment:
Could not find docs for `device.get_gpus`.
Can you:
1. Ask middleware dev that was working on this, to add at least some documentation.
2. Yourself add GpuDevice interface with what you see in inspector.

Reply:
`device.get_gpus` is not a public method. Private methods don't have their documentation in `api/docs`. An alternative is `device.get_info` which has the same purpose but less specific according to the middleware dev.

Reply:
@sonicaj UI should not be using private methods. Why get_info is less specific?

Reply:
`get_info `is less specific in the sense that it addresses more than just GPUs whereas `get_gpus `only addresses GPUs. Both return the same information when `get_info `is used to get gpu devices info.